cmd/knoxite/store.go

```diff
@@ -81,22 +100,29 @@ func store(repository *knoxite.Repository, chunkIndex *knoxite.ChunkIndex, snaps
 		return gerr
 	}
 
-	if uint(len(repository.BackendManager().Backends))-opts.FailureTolerance <= 0 {
+	if len(repository.BackendManager().Backends)-int(opts.FailureTolerance) <= 0 {
 		return ErrRedundancyAmount
 	}
-	compression, err := CompressionTypeFromString(opts.Compression)
+	compression, err := utils.CompressionTypeFromString(opts.Compression)
 	if err != nil {
 		return err
 	}
-	encryption, err := EncryptionTypeFromString(opts.Encryption)
+	encryption, err := utils.EncryptionTypeFromString(opts.Encryption)
 	if err != nil {
 		return err
 	}
 
+	var tol uint
+	if inttol := len(repository.BackendManager().Backends) - int(opts.FailureTolerance); inttol <= 0 {
```

Comment:
I don't think we can ever reach this condition, as we already check for it above.

Reply:
You're right - I've changed this line to just convert the value to `uint` for tol